atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/CheckAndSetException.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.keyvalue.api;
+
+import java.util.List;
+
+public class CheckAndSetException extends RuntimeException {
```

Comment:
I'm a bit confused because this now seems to create an exception with a message string with the data exposed.  Seemed like we had reached the conclusion that it wasn't safe to log?